app/src/main/kotlin/by.liauko.siarhei.fcc/repository/RepositoryUtil.kt

```diff
@@ -1,12 +1,18 @@
 package by.liauko.siarhei.fcc.repository
 
 import android.content.Context
+import android.os.AsyncTask
+import by.liauko.siarhei.fcc.database.CarLogDatabase
+import by.liauko.siarhei.fcc.database.entity.AppEntity
+import by.liauko.siarhei.fcc.database.entity.FuelConsumptionEntity
+import by.liauko.siarhei.fcc.database.entity.LogEntity
 import by.liauko.siarhei.fcc.util.ApplicationUtil.dataPeriod
 import by.liauko.siarhei.fcc.util.DataPeriod
 import by.liauko.siarhei.fcc.util.DataType
 import java.util.Calendar
 
 object RepositoryUtil {
```

Comment:
Not sure I get why the object exists. You could just leave the function to top-level declared and that's it.